apps/cyberstorm-storybook/public/css/globals.css

```diff
@@ -0,0 +1,143 @@
+  --color-green--40: #40AE7F;
+  --color-green--30: #30825F;
+  --color-green--20: #205740;
+  --color-green--10: #102B20;
+
+  --color-yellow--90: #FDF5DC;
+  --color-yellow--80: #FAECB9;
+  --color-yellow--70: #F8E295;
+  --color-yellow--60: #F5D972;
+  --color-yellow--50: #F3CF4F;
+  --color-yellow--40: #C2A63F;
+  --color-yellow--30: #927C2F;
+  --color-yellow--20: #615320;
+  --color-yellow--10: #312910;
+
+  --color-orange--90: #FDE1D9;
+  --color-orange--80: #FBC4B2;
+  --color-orange--70: #FAA68C;
+  --color-orange--60: #F88965;
+  --color-orange--50: #F66B3F;
+  --color-orange--40: #C55632;
+  --color-orange--30: #944026;
+  --color-orange--20: #622B19;
+  --color-orange--10: #31150D;
+
+  --color-pink--90: #F9DDF6;
+  --color-pink--80: #F2BBED;
+  --color-pink--70: #EC9AE3;
+  --color-pink--60: #E578DA;
+  --color-pink--50: #DF56D1;
+  --color-pink--40: #B245A7;
+  --color-pink--30: #86347D;
+  --color-pink--20: #592254;
+  --color-pink--10: #2D112A;
+
+  --color-red--90: #FCD7DE;
+  --color-red--80: #F9AFBD;
+  --color-red--70: #F7889C;
+  --color-red--60: #F4607B;
+  --color-red--50: #F1385A;
+  --color-red--40: #C12D48;
+  --color-red--30: #912236;
+  --color-red--20: #601624;
+  --color-red--10: #300B12;
+
+  --color-cyber-green: var(--color-cyber-green--50);
+  --color-purple: var(--color-purple--50);
+  --color-blue: var(--color-blue--50);
+  --color-green: var(--color-green--50);
+  --color-yellow: var(--color-yellow--50);
+  --color-orange: var(--color-orange--50);
+  --color-pink: var(--color-pink--50);
+  --color-red: var(--color-red--50);
+
+  --color-surface--1: var(--color-default-purple--1);
+  --color-surface--2: var(--color-default-purple--2);
+  --color-surface--3: var(--color-default-purple--3);
+  --color-surface--4: var(--color-default-purple--4);
+  --color-surface--5: var(--color-default-purple--5);
+  --color-border: var(--color-default-purple--6);
+  --color-border-2: var(--color-default-purple--7);
+  --color-border-3: var(--color-default-purple--8);
+
+  --color-gradient-blue-green--darker: linear-gradient(283.56deg, #062131 0%, #102B20 100%);
+  --color-gradient-blue-green--dark: linear-gradient(283.56deg, #0B4162 0%, #205740 100%);
+  --color-gradient-blue-green: linear-gradient(283.56deg, #1CA3F5 0%, #23FFB0 100%);
+  --color-gradient-blue-green--light: linear-gradient(283.56deg, #49B5F7 0%, #73E1B2 100%);
+  --color-gradient-purple-green--darker: linear-gradient(283.56deg, #1F0A2E 0%, #102B20 100%);
+  --color-gradient-purple-green--dark: linear-gradient(283.56deg, #2F0E44 0%, #0A4C35 100%);
+  --color-gradient-purple-green: linear-gradient(283.56deg, #9D30E4 0%, #23FFB0 100%);
+  --color-gradient-purple-green--light: linear-gradient(283.56deg, #B159E9 0%, #73E1B2 100%);
+
+
+  --size-medium: 1rem;
+  --size-medium-small: 0.625rem;
+  --size-small: 0.5rem;
+
+  --gap-medium: var(--size-medium);
+
+  --padding-medium: var(--size-medium);
+  --padding-medium-small: var(--size-medium-small);
+}
+
+.label {
```

Comment:
I guess this is for the button component's text. I'd argue that the text doesn't require a wrapper div, If it does, this should be defined in button's CSS module and not as a global. If we're absolutely sure it should be global, it should be named more descriptively or include a comment describing what's it for. But I'd argue that a random developer won't find the class from here and thus it won't be used globally.

Also neither font-size or line-height should use pixels, I guess?

Reply:
Ideally each of the button's children would be wrapped in something and have their own classes to easily hook on if needed and to avoid having to resort to chaining and nesting of classes. But we would need to establish guidelines on how to keep it consistent.

In any case, yes, the classes should reside in the component itself.

Reply:
label moved to the button module, unused padding thingy removed, pixels swithced to rem.